bm-persona/Fitness/FitnessDataSource/Gym.swift

```diff
@@ -43,8 +47,8 @@ class Gym: SearchItem {
     
     var openingTimeToday: Date? = nil
     var closingTimeToday: Date? = nil
-    var latitude: Double? = nil
-    var longitude: Double? = nil
+    var latitude: Double = Double.nan
```

Comment:
Was gonna adjust SearchItem protocol in future commit to make certain fields like this one optional

Reply:
I made a HasLocation protocol that requires latitude and longitude to exist and Gym implements it, so I changed lat and long here to not be optional.